src/features/user/Account/EditModal.tsx

```diff
@@ -6,23 +6,24 @@ import { UserPropsContext } from '../../common/Layout/UserPropsContext';
 import MaterialTextField from '../../common/InputTypes/MaterialTextField';
 import styles from './AccountHistory.module.scss';
 import {
-  Backdrop,
   CircularProgress,
+  Modal,
```

Comment:
There are minor changes seen in height/width/font styles/spacing for the EditModal as a whole.
![Screenshot 2022-05-05 at 5 26 48 PM](https://user-images.githubusercontent.com/44917347/166918306-dd74c0d8-9543-4d6b-953c-9a4fac7831fa.png)

Reply:
@mohitb35 Please verify the zoom level 
The left window seems to e more zoomed-in than right

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r867877084

Reply:
Fixed in #1467